src/js/components/AppPageComponent.jsx

```diff
@@ -273,15 +282,148 @@ var AppPageComponent = React.createClass({
     );
   },
 
+  getVolumeStatus: function () {
+    var volume = this.getVolume();
+
+    if (volume == null) {
+      return null;
+    }
+
+    var className = classNames("volume-status", {
+      "volume-attached": volume.status === "Attached"
+    });
+
+    return (
+      <h2 className={className}>{volume.status}</h2>
+    );
+  },
+
+  getVolume: function () {
```

Comment:
Looks like this belongs to something like a store. AppsStore directly below getTasks() ? Separation of concerns.